src/bundle/Form/Locations/Actions.php

```diff
@@ -17,6 +18,8 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('delete', SubmitType::class)
+            ->add('add', SubmitType::class)
+            ->add('parentLocationId', HiddenType::class)
```

Comment:
for now it's can even be a TextType, that way you can test it manually (it's specified like that  in https://jira.ez.no/browse/EZP-27503 see attachment)